microsoft-edge/devtools-guide-chromium/console/utilities.md

```diff
@@ -25,942 +25,801 @@ ms.date: 05/04/2021
+## Query selector all: \$\$(selector, \[startNode\])
 
-```javascript
-$$(selector, [startNode])
-```
+<!-- summary bubbled up to Summary table: -->
+`$$()` returns a list of elements that match the specified CSS selector, like `document.querySelectorAll()`.
 
-### Example
+<!-- add'l info -->
+This function is similar to [document.querySelectorAll()](https://developer.mozilla.org/docs/Web/API/Document/querySelectorAll) but returns an `array` of element, instead of a `NodeList`.
 
-In the following example, `$$()` creates an array of all `<img>` elements in the current webpage, and displays the value of the `src` property for each element:
+#### Syntax
 
 ```javascript
-var images = $$('img');
-for (each in images) {
-    console.log(images[each].src);
-}
+$$(selector, [parentElement])
 ```
 
-Result:
+#### Example 1
 
-![Using $$() to select all images in the webpage and display the sources](./utilities-images/console-element-selector-image-all.png)
+1. Press **Ctrl+Shift+J** (Windows, Linux) or **Command+Option+J** (macOS).  The **Console** tool opens in DevTools, next to the webpage.
 
-This query selector function also supports a second parameter, `startNode`, that specifies an element or node from which to search for elements.  The default value of the parameter is `document`.
+1. In the **Console** tool, type `$$('button')`. All the `<button>` elements in the webpage are displayed:
 
-### Example
+   ![The result of $$('button')](./utilities-images/element-selector-image-all.png)
 
-The following, modified version of the previous example uses `$$()` to create an array of all `<img>` elements that appear in the current webpage after the selected node:
+#### Example 2
 
-```javascript
-var images = $$('img', document.querySelector(`title--image`));
-for (each in images) {
-   console.log(images[each].src);
-}
-```
-
-Here's the result.  `$$()` selects all images that appear after the specified `<div>` element in the webpage, and displays the sources:
+This `$$()` function also supports a second parameter, `parentElement`, that specifies a parent element which to search in. The default value of the parameter is `document`, which means that the entire webpage is searched by default. By specifying a parent element, you can search for elements within a specific part of the webpage.
 
-![Using $$() to select all images that appear after the specified <div> element in the webpage and display the sources](./utilities-images/console-element-selector-image-filter-all.png)
+1. Press **Ctrl+Shift+J** (Windows, Linux) or **Command+Option+J** (macOS).  The **Console** tool opens in DevTools, next to the webpage.
 
+1. In the **Console** tool, type `$$('button', $('li.task'))`. All the `<button>` elements that are descendants of the first `<li class="task">` element are displayed:
 
-###### Entering a newline character 
+   ![The result of $$('button', $('li.task'))](./utilities-images/element-selector-image-all-descendants.png)
```

Comment:
is it possible to show a Delete button in webpage on left, next to task 2 & task 1?

rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/utilities?branch=pr-en-us-2936#example-2-2